packages/destination-actions/src/destinations/salesforce/sf-operations.ts

```diff
@@ -23,19 +24,11 @@ export default class Salesforce {
   }
 
   createRecord = async (payload: GenericPayload, sobject: string) => {
+    const JSON_data = this.buildJSONData(payload, sobject)
```

Comment:
```suggestion
    const json = this.buildJSONData(payload, sobject)
```

At Segment, unusual to have snake case in variable names